xrpl/models/base_model.py

```diff
@@ -73,6 +73,28 @@ def _value_to_json(value: XRPL_VALUE_TYPE) -> XRPL_VALUE_TYPE:
     return value
 
 
+def process_xrpl_json(value: Union[str, Dict[str, Any]]) -> Dict[str, Any]:
```

Comment:
I'd recommend making this a helper class method, instead of a separate function - it saves you some imports. Similar to `_get_only_init_args`.

Reply:
okay, I have done this. But I dislike the fact that the `self/cls` argument is not useful inside the `process_xrpl_json` function. I'm only using it to hook into this function

Reply:
I think the tradeoff is worth it given that it makes it easier to use, but if someone strongly objects I'll withdraw

Reply:
yeah, I think it's a reasonable tradeoff 👍